From 95118e921bed4e51da8c013b2dedfcaae6ab5dfc Mon Sep 17 00:00:00 2001
From: Axel Haslam <axelhaslam@ti.com>
Date: Thu, 3 Mar 2011 11:49:35 -0600
Subject: OMAP4: Dont  enable PMD clocks at boot

to save power, This patch changes the
android defconfig so that PMD clocks
are not enabled by default at bootup.

Change-Id: Ia2c554e535b7e58b75a0231199808bbb114c9725
Signed-off-by: Axel Haslam <axelhaslam@ti.com>
---
 arch/arm/configs/android_4430_defconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/configs/android_4430_defconfig b/arch/arm/configs/android_4430_defconfig
index 45078ec..76256a4 100644
--- a/arch/arm/configs/android_4430_defconfig
+++ b/arch/arm/configs/android_4430_defconfig
@@ -304,7 +304,7 @@ CONFIG_MACH_OMAP4_PANDA=y
 # CONFIG_INTERCONNECT_IO_POSTING is not set
 # CONFIG_ENABLE_L3_ERRORS is not set
 CONFIG_OMAP_ALLOW_OSWR=y
-CONFIG_OMAP4_PMD_CLKS_ENABLE=y
+# CONFIG_OMAP4_PMD_CLKS_ENABLE is not set
 # CONFIG_OMAP4_KEEP_STATIC_DEPENDENCIES is not set
 
 #
-- 
1.6.0.4

